tests/run/assets/kwargs.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+def func(context, x, **kwargs):
+    context.logger.info(x)
+    context.logger.info(kwargs)
+    if not kwargs:
```

Comment:
you can log the kwargs as result and then assert the result in the test instead of just checking if you got kwargs for better coverage